vyper.asciidoc

```diff
@@ -175,30 +260,27 @@ Smart contracts can write data to two places, Ethereum's global state trie or Et
 
-Log:: As previously mentioned, a smart contract can also write to Ethereum's chain data through log events. While Vyper initially employed the pass:[__]logpass:[__] syntax for declaring these events, an update has been made which brings Vyper's event declaration more in line with Solidity's original syntax. For example, Vyper's declaration of an event called MyLog was originally `MyLog: pass:[__]logpass:[__]({arg1: indexed(bytes[3])})` Vyper's syntax has now become `MyLog: event({arg1: indexed(bytes[3])})`. It is important to note that the execution of the log event in Vyper was and still is as follows `log.MyLog("123")`.
+Log:: As previously mentioned, a smart contract can also write to Ethereum's chain data through log events. While Vyper initially employed the pass:[__]logpass:[__] syntax for declaring these events, an update has been made which brings Vyper's event declaration more in line with Solidity's original syntax. For example, Vyper's declaration of an event called MyLog was originally `MyLog: pass:[__]logpass:[__]({arg1: indexed(bytes[3])})` Vyper's syntax has now become `MyLog: event({arg1: indexed(bytes[3])})`. It is important to note that the execution of the log event in Vyper was, and still is, as follows `log.MyLog("123")`.
 
 While smart contracts can write to Ethereum's chain data (through log events), smart contracts are unable to read the on-chain log events, which they created. Notwithstanding, one of the advantages of writing to Ethereum's chain data via log events is that logs can be discovered and read, on the public chain, by light clients. For example, the logsBloom value in a mined block can indicate whether or not a log event was present. Once this has been established the log data can be obtained through the path of logs -> data inside a given transaction receipt.
 
 :revnumber: v1.1
 :revdate: 2018-04-10 (yyyy-mm-dd)
-:revremark: Just want to keep an eye on the development in Vyper GitHub in order to compare and contrast the differences between Vyper and Solidity's ERC20 Token Interface Implementation closer to ethereumbook being published.
-:revstatus: TODO
+:revremark: This section is now complete
 
 [[erc20_token_interface_implementation_sec]]
 === ERC20 token interface implementation
-Vyper has implemented ERC20 as a precompiled contract and allows to use it by default.
-Contracts in Vyper must be declared as global variables. An example for declaring the ERC20 variable can be
-token: address(ERC20).
+Vyper has implemented ERC20 as a precompiled contract; allowing these smart contracts to be easily used by default. Contracts in Vyper must be declared as global variables. An example for declaring the ERC20 variable is as follows.
```

Comment:
Just a note here: this isn't fully pre-compiled contract. But just an interface to consume another ERC20 contract ;)